src/resources/lang/en/notifications.php

```diff
@@ -49,4 +49,6 @@
 
     'setup_label'             => 'Notifications Setup',
     'setup_description'       => 'Allow user to define notification channels and subscriptions.',
+
+    'test_integration'        => 'Test Integration',
```

Comment:
```suggestion
    'test_integration'        => 'Test',
```

Isn't "Test" enough ? Caption looks a bit too long on your screen - especially for mobiles
Also, I wonder if we shouldn't add icons to buttons ?